jenkins/databricks/test.sh

```diff
@@ -164,6 +165,23 @@ DELTA_LAKE_CONFS="--driver-memory 2g"
 export PYSP_TEST_spark_eventLog_enabled=true
 mkdir -p /tmp/spark-events
 
+rapids_shuffle_smoke_test() {
+    echo "Run rapids_shuffle_smoke_test..."
+
+    # using MULTITHREADED shuffle
+    PYSP_TEST_spark_rapids_shuffle_mode=MULTITHREADED \
+    PYSP_TEST_spark_rapids_shuffle_multiThreaded_writer_threads=2 \
+    PYSP_TEST_spark_rapids_shuffle_multiThreaded_reader_threads=2 \
+    PYSP_TEST_spark_cores_max=2 \
+    PYSP_TEST_spark_executor_cores=1 \
```

Comment:
we are running this in local mode with all cores on box then, so we are actually getting multiple executors?  would be good to add comment about why configs are set this way

Reply:
So even in local mode, we should be able to see the MT shuffle start up and write with 2 threads and read with the same.

I would remove spark.cores.max, spark.executor.cores and the gpu memory configs below (min/max/allocFraction), this isn't an issue here since we are not sharing the GPU, we are the only process running.